content/guides/building/port_status.html

```diff
@@ -51,7 +57,16 @@
 	<td><img src='/files/status-icons/no-work.png' alt='No Work Complete'></td>
 	<td><img src='/files/status-icons/no-work.png' alt='No Work Complete'></td>
 	<td>0%</td></tr>
+</table>
 
+<table>
+<tr><td COLSPAN=7><h3>RISC-V (Tier 2)</h3>An open-source computer architecture with many implementations. **Note that real RISC-V hardware is not yet supported. There are issues with some packages and libstdc++ is partially broken. Additionally, nightly builds are not yet available.**</td></tr>
+<tr><th>Platform</th><th>Interest</th><th>Target</th><th>Haiku Loader</th><th>Haiku Kernel</th><th>Application Server</th><th>Status</th></tr>
+<tr><td>TinyEMU (raw platform)</td><td>High</td><td>x86_gcc2</td><td><img src='/files/status-icons/complete.png' alt='Complete'></td><td><img src='/files/status-icons/complete.png' alt='Complete'></td><td><img src='/files/status-icons/complete.png' alt='Complete'></td><td>Under Development</td></tr>
```

Comment:
Why use x86_gcc2, x86 as targets? Both TinyEMU and QEMU are cross platform and can run on many host architectures and operating systems (32 and 64 bit x86, Haiku and openSUSE Linux are confirmed). Haiku internal target name for RISC-V is riscv64.

Reply:
@X547 Sorry, I forgot to edit that part of the table! I'll fix it now.